### ISSUE TITLE
Polyfill for __webpack_require__ can break third-party code

### ISSUE THREAD

**Issue by seanf** [sormy/webpack-fix-default-import-plugin#3]
I'm sure this is no great surprise to you, and probably unavoidable, but I guess it could be useful as a "known issues" type thing.

The color-convert library (as used by chartjs-color) iterates through all the exports in a module (conversions.js), but it doesn't like it when it bumps into the new `default` entry.

```
VM85667 index.js:19 Uncaught TypeError: Cannot read property '1' of null
    at eval (VM85667 index.js:19)
    at Object.<anonymous> (VM84139 frontend.js:21969)
    at __webpack_require__ (VM84139 frontend.js:20)
    at eval (VM85666 index.js:2)
    at Object.<anonymous> (VM84139 frontend.js:8323)
    at __webpack_require__ (VM84139 frontend.js:20)
    at eval (VM85665 core.helpers.js:5)
    at Object.<anonymous> (VM84139 frontend.js:21958)
    at __webpack_require__ (VM84139 frontend.js:20)
    at eval (VM85663 chart.js:6)
```

That bit of code looks like this at runtime:

```js
var conversions = __webpack_require__(/*! ./conversions */ 1913);
...
for (var func in conversions) {
  var pair = /(\w+)2(\w+)/.exec(func),
      from = pair[1],
      to = pair[2];
```

**Comment by sormy** [comment]
Thanks for note. I will update README.

**Comment by sormy** [comment]
Please use `esModuleInterop` option that is available since TypeScript 2.7 to use default imports in the same way as in Babel.
